### DISCUSSION TITLE
Expand page columns with additional information

### DISCUSSION THREAD

**Discussion by haroldvandekamp** [WikiTransformationProject/wikitraccs-releases#60] [Ideas]
After a page (or blog) has been migrated, the page in SharePoint contains valuable data regarding the migrated page. Can additional properties be added to make it easier to calculate how much work has to be done manually after the automatic migration is finished? I'm thinking of the following additional information:
- Total macros on page that could not be migrated
- List of the macros that could not be migrated
- Total page or file not found links
- Total referenced people that could not be matched

With that data makes it also easier to pinpoint which pages would a large amount of time to finalize with the manual cleanup work.

**Comment by heinrich-ulbricht** [comment]
@haroldvandekamp Did you have a look at the [existing metadata for pages](https://www.wikitransformationproject.com/blog/2022/10/22/measuring-page-migration-success/)? Some information should already be there, like links and users that could not be resolved, and some information about macros.

I need to provide a more detailed documentation about the existing columns, then we can check what is missing.

**Reply by haroldvandekamp** [reply]
Thank you referencing that documentation @heinrich-ulbricht . I missed that one. I will dive into that next week.

**Reply by heinrich-ulbricht** [reply]
No need to hurry, you are fast enough 😁
